app/controllers/pages_controller.rb

```diff
@@ -1,4 +1,6 @@
 class PagesController < ApplicationController
-  def home
+  skip_before_action :authenticate_user!, only: :bikes
+
+  def bikes
```

Comment:
Can you delete all in pagesController please as it is useless

Reply:
It's done, thanks !